plugins/vpp/ifplugin/vppcalls/dump_nat_vppcalls.go

```diff
@@ -144,6 +150,44 @@ func (handler *natVppHandler) nat44AddressDump() (addresses []*nat.Nat44Global_A
 	return
 }
 
+// virtualReassemblyDump returns current NAT44 virtual-reassembly configuration. The output config may be nil.
+func (handler *natVppHandler) virtualReassemblyDump() (vrIPv4 *nat.Nat44Global_VirtualReassemblyIPv4, vrIPv6 *nat.Nat44Global_VirtualReassemblyIPv6, err error) {
+	defer func(t time.Time) {
+		handler.stopwatch.TimeLog(bin_api.NatGetReass{}).LogTimeEntry(time.Since(t))
+	}(time.Now())
+
+	req := &bin_api.NatGetReass{}
+	reply := &bin_api.NatGetReassReply{}
+
+	if err := handler.dumpChannel.SendRequest(req).ReceiveReply(reply); err != nil {
+		return nil, nil, fmt.Errorf("failed to get NAT44 virtual reassembly configuration: %v", err)
+	}
+	if reply.Retval != 0 {
+		return nil, nil, fmt.Errorf("%s returned %d", reply.GetMessageName(), reply.Retval)
+	}
+
+	// IPv4
+	if !(reply.IP4MaxReass == MaxReassembly && reply.IP4MaxFrag == MaxFragments && reply.IP4Timeout == Timeout && reply.IP4DropFrag == 0) {
```

Comment:
This comparison to some defaults should be handled in caller of vppcalls handler. We should keep the vppcalls handlers as low-level as possible, so we can always dump real state of VPP.